website/content/docs/secrets/databases/redis.mdx

```diff
@@ -0,0 +1,124 @@
+---
+layout: docs
+page_title: Redis - Database - Secrets Engines
+description: |-
+  Redis is one of the supported plugins for the database secrets engine.
+  This plugin generates database credentials dynamically based on configured
+  roles for the Redis database.
+---
+
+# Redis Database Secrets Engine
+
+Redis is one of the supported plugins for the database secrets engine. This
+plugin generates database credentials dynamically based on configured roles for
+the Redis database.
+
+See the [database secrets engine](/docs/secrets/databases) docs for
+more information about setting up the database secrets engine.
+
+## Capabilities
+
+| Plugin Name                 | Root Credential Rotation | Dynamic Roles | Static Roles | Username Customization |
+| --------------------------- | ------------------------ | ------------- | ------------ | ---------------------- |
+| `redis-database-plugin`     | Yes                      | Yes           | Yes          | No                     |
+
+## Setup
+
+1.  Enable the database secrets engine if it is not already enabled:
+
+    ```shell-session
+    $ vault secrets enable database
+    Success! Enabled the database secrets engine at: database/
+    ```
+
+    By default, the secrets engine will enable at the name of the engine. To
+    enable the secrets engine at a different path, use the `-path` argument.
+
+1.  Configure Vault with the proper plugin and connection configuration:
+
+    ```shell-session
+    $ vault write database/config/my-redis-database \
+      plugin_name="redis-database-plugin" \
+      host="localhost" \
+      port=6379 \
+      username="vaultuser" \
+      password="vaultpass" \
+      allowed_roles="my-*-role"
+    ```
```

Comment:
Should we re-use the same sample values from the api-doc page for consistency and to let users see the relation between both methods?

Reply:
👍